src/libs/OptionsListUtils.js

```diff
@@ -159,17 +161,20 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
     searchValue = '',
     showChatPreviewLine = false,
     showReportsWithNoComments = false,
+    hideReadReports = false,
+    sortByAlphaAsc = false,
 }) {
     let recentReportOptions = [];
     const pinnedReportOptions = [];
     const personalDetailsOptions = [];
 
     const reportMapForLogins = {};
-    const orderedReports = lodashOrderBy(reports, [
-        sortByLastMessageTimestamp
-            ? 'lastMessageTimestamp'
-            : 'lastVisitedTimestamp',
-    ], ['desc']);
+    let iteratees = sortByLastMessageTimestamp
```

Comment:
NAB, (not a blocker) `sortProperty` rolls off the tongue a bit nicer than `iteratees`